.github/CODEOWNERS

```diff
@@ -31,7 +31,7 @@
 
 
 # Android team
-/core/domain/android_validation_constants*.py @BenHenning
+/core/domain/android_validation_constants*.py @seanlip  # After 16 August 2020, restore @BenHenning
```

Comment:
IIRC, Inline comments don't work on CODEOWNER files (Compilation will fail and code owner functionality won't work!). You can move this comment to line 32*

Reply:
+1 I think I've seen errors before arising from this. PTAL @BenHenning

Reply:
Note: Please follow what @brianrodri did in this PR: https://github.com/oppia/oppia/pull/9893/files (with TODOs, comments on their own line, etc.).

Also, could you please add a note at the top of the file explaining the procedure for this sort of situation, and linking to that PR as canonical? Hopefully this will help avoid future issues like this.

Reply:
Done!